SunEngine.Tests/DefaultInit.cs

```diff
@@ -0,0 +1,22 @@
+﻿using SunEngine.Core.DataBase;
+
+namespace SunEngine.Tests
+{
+    internal static class DefaultInit
+    {
+        private static string connectionString = "Host=localhost;Database=SunEngine;Username=postgres;Password=password";
+        private static string provider = "PostgreSQL.9.5";
```

Comment:
Is it in memory database?

Reply:
linq2db not provide in-memory options

Reply:
Got it. Will it work on my PC without additional installations? I believe Postgres requires additional setup.